gateway.yml

```diff
@@ -0,0 +1,3 @@
+baseUri: http://0.0.0.0/api/
```

Comment:
Should this be `http://0.0.0.0:8080/radar-gateway/` ?

Reply:
And mp should be at `http://managementportal-app:8090/managementportal`

Reply:
thanks! updated